server/serializers/subscriptions.go

```diff
@@ -57,6 +60,37 @@ type CreateSubscriptionRequestPayload struct {
 	TargetBranch   string `json:"targetBranch"`
 }
 
+type GetSubscriptionFilterPossibleValuesRequestPayload struct {
+	Organization string   `json:"organization"`
+	ProjectID    string   `json:"projectId"`
+	EventType    string   `json:"eventType"`
+	Filters      []string `json:"filters"`
+}
+
+type SubscriptionFilter struct {
+	InputID string `json:"inputId"`
+}
+
+type GetSubscriptionFilterValuesRequestPayloadFromClient struct {
+	Subscription *CreateSubscriptionBodyPayload `json:"subscription"`
+	InputValues  []SubscriptionFilter           `json:"inputValues"`
```

Comment:
```suggestion
	InputValues  []*SubscriptionFilter           `json:"inputValues"`
```
Wherever we are using array of struct, we should use pointers.